styles/base/index.js

```diff
@@ -0,0 +1,20 @@
+var fs = require('fs')
+var path = require('path')
+var compile = require('micro-css')
+var result = ''
+var additional = ''
+
+fs.readdirSync(__dirname).forEach(function (file) {
+  if (/\.mcss$/i.test(file)) {
+    result += fs.readFileSync(path.resolve(__dirname, file), 'utf8') + '\n'
+  }
+
+  if (/\.css$/i.test(file)) {
+    additional += fs.readFileSync(path.resolve(__dirname, file), 'utf8') + '\n'
+  }
+})
+
+const flatpickr = require.resolve('flatpickr/dist/flatpickr.css')
```

Comment:
Oh, these flatpickr styles will now be needed in the `light` and `dark` themes now that you are no longer merging this file in. In fact base/index.js can now be deleted.

It is used by the gatherings editor when clicking the date field. Without these additional styles, the date picker is not visible.

Reply:
Ha! Thanks, I didn't really consider how useless `base/index.js` would be with this solution. Should be resolved now.